packages/payment-integration-api/src/payment/payment.ts

```diff
@@ -78,12 +78,6 @@ export interface WithBankAccountInstrument {
     firstName?: string;
     lastName?: string;
     businessName?: string;
-    address1: string;
-    address2: string;
-    city: string;
-    countryCode: string;
-    postalCode: string;
-    stateOrProvinceCode: string;
     shouldSaveInstrument?: boolean;
```

Comment:
Can this potentially break other integrations?

Reply:
@animesh1987 no, this is only for Braintree ACH payment method

Reply:
Just a note for future, if this is only used by braintree shall we move this in a braintree package?